src/schema/users.ts

```diff
@@ -1905,6 +1925,11 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       try {
         const updatedUser = { ...user, ...data, image: avatar };
         updatedUser.email = updatedUser.email?.toLowerCase();
+
+        if ('showPlusGift' in data) {
+          updatedUser.flags.showPlusGift = data.showPlusGift;
+        }
```

Comment:
With above changes to accept flags we can update flags with `updateFlagsStatement` directly

Reply:
Could not use updateFlagsStatement because in save it is not possible, but I'm updating the whole flags merging with old ones